release-notes.txt

```diff
@@ -1,5 +1,540 @@
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release
+==================
+
+The 2024.07 release, nicknamed "StrikinglyCrowded", includes many features and bug fixes.
+Here are some of the highlights:
+
++ RIOT now boots on the Arduino Nano 33 BLE Sense (#20668)
++ To support boards with Ethernet plugs such as the Adafruit Ethernet FeatherWing, RIOT now ships a driver for the W5500 ethernet chip (#20301)
++ RIOT now leverages the ARMv8-M hardware support to enforce the stack size (#20633)
++ The PSA Crypto API implementation in RIOT now supports persistent key storage and additional hashing algorithms (#20099, #20598, #20698)
++ This release features several useful additions to the RIOT CoAP stacks: a coaps forward proxy for gcoap, delayed responses for nanocoap_sock, and the ability to mount a remote filesystem over CoAP (#20454, #20266, #20687)
+* Support of the 16-bit MSP430 MCU has been improving over the last few releases, with the addition of power management in this release as an example (#20613)
+* First steps towards enabling RIOT support of new Microchip SAM MCUs (#20708, #20710, #20713, #20714)
+* Users of BUILD_IN_DOCKER=1 will now be warned if they are using an outdated docker container (#20472)
+
+
+175 pull requests, composed of 358 commits, have been merged since the
+last release, and 5 issues have been solved. 36 people contributed with
+code in 80 days. 650 files have been touched with 88156 (+) insertions and
+3949 deletions (-).
+
+
+Notations used below
+====================
+
+    + means new feature/item
+    * means modified feature/item
+    - means removed feature/item
+
+
+New features and changes
+========================
+
+Core
+----
+
+* core/assert: halt running thread instead of panic (#20627)
+* core/lib/assert: consistify output and cleanup documentation (#20701)
+* core/panic: make reboot on panic configurable (#20630)
+* core/thread: "fix" valgrind errors in thread_measure_stack_free() (#20726)
+
+System Libraries
+----------------
+
++ sys/event.h: Fixed documentation error by adding a type cast. (#20704)
++ sys/shell: add support for running a batch of commands from a file (#20651)
+* rust: bump rust-riot-{sys,wrappers} version (#20786)
+* shell/vfs: Handle print failure in genfile cmd gracefully (#20595)
+* sys/hashes: change SHA2 PAD type to const (#20729)
+* sys/psa_crypto: ed25519 private key {ex,im}port (#20334)
+* sys/psa_crypto: Implement persistent key storage (#20099)
+* sys/psa_crypto: remove keysize determination from Kconfig files (#20674)
+* sys/psa_crypto: SHA-{384,512/{224,256}} (#20598)
+* sys/psa_crypto: sha3 support (#20698)
+* sys/shell: Update cryptoauthlib shell commands (#20673)
+* sys/suit: return error when URL buffer is too small [backport
+  2024.01] (#20563)
+* sys/vfs: use atomic_utils rather C11 atomics (#20513)
+* treewide: Rust updates [backport 2024.01] (#20384)
```

Comment:
Same thing here: 20303

Reply:
And same here, too. I wonder why they ended up in the automatically generated changelog.